tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -198,3 +244,45 @@ def _test_data_repository_task(remote_command_executor, mount_dir, bucket_name,
     assert_that(file_owner).is_equal_to("6666")
     assert_that(file_group).is_equal_to("6666")
     assert_that(file_permissions).is_equal_to("0100777")
+
+
+def create_backup_test_file(remote_command_executor, mount_dir):
+    logging.info("Creating a backup test file in fsx lustre mount directory")
+    sge_commands = SgeCommands(remote_command_executor)
+    remote_command_executor.run_remote_command(
+        "echo 'FSx Lustre Backup test file' > {mount_dir}/file_to_backup".format(mount_dir=mount_dir)
+    )
+    job_command = "cat {mount_dir}/file_to_backup ".format(mount_dir=mount_dir)
+    result = sge_commands.submit_command(job_command)
+    job_id = sge_commands.assert_job_submitted(result.stdout)
+    sge_commands.wait_job_completed(job_id)
+    sge_commands.assert_job_succeeded(job_id)
+    result = remote_command_executor.run_remote_command("cat {mount_dir}/file_to_backup".format(mount_dir=mount_dir))
+    assert_that(result.stdout).is_equal_to("FSx Lustre Backup test file")
```

Comment:
What's the purpose of the last two lines?

Reply:
It is just to assert that the file exists after the sge job is completed. This check ensures that the original cluster had this file in the shared_dir with the content `FSx Lustre Backup test file`.